README.md

```diff
@@ -2,6 +2,8 @@
 
 The Operator Courier is used to build, validate and push Operator Artifacts.
 
+We currently support Python 3.6 and above.
```

Comment:
Instead of using `We` here, can we use `Operator-Courier` as the subject of the sentence?

`Operator Courier is currently supported on Python 3.6 and above`

Reply:
Thanks @kevinrizza. Updated.